src/main/kotlin/com/autonomousapps/model/declaration/Configurations.kt

```diff
@@ -79,13 +79,16 @@ internal object Configurations {
   }
 
   @OptIn(ExperimentalStdlibApi::class)
-  private fun findVariant(variantSlug: String, supportedSourceSets: Set<String>): Variant? {
+  private fun findVariant(variantSlug: String, supportedSourceSets: Set<String>, hasCustomSourceSets: Boolean): Variant? {
```

Comment:
I added the `hasCustomSourceSets` flag here so that we know if we need to identify "android test source sets" or not. Otherwise a custom source set like "testIntegration" in a standard JVM project might be classified as Android Test.